src/main.rs

```diff
@@ -572,3 +572,19 @@ fn install_tracing(filter_directives: &str) -> Result<()> {
 
     Ok(())
 }
+
+fn install_color_eyre() -> Result<()> {
+    color_eyre::config::HookBuilder::new()
+        // Don't display the backtrace reminder by default:
+        //   Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
+        //   Run with RUST_BACKTRACE=full to include source snippets.
+        // Backtraces can be enabled with the `--backtrace` switch or the `RUST_BACKTRACE`
+        // environment variable (documented in the `--help` output).
+        .display_env_section(false)
+        // Don't display location information by default:
+        //   Location:
+        //      src/steps.rs:92
+        // Location information is displayed in the backtrace, but usually isn't needed anyways.
+        .display_location_section(false)
```

Comment:
The backtrace reminder section is indeed useless to the end user and should be removed.

For the location section, I kinda think that it should be retained as without attribute `debug = true` enabled, the location info given in BACKTRACE isn't that useful and makes us difficult to locate the root cause:

> Execute `topgrade` then precess Ctrl-C to interrupt it, I have `pre_sudo` set in my configuration file

```
$ topgrade

── 15:26:12 - Sudo ─────────────────────────────────────────────────────────────
[sudo] password for steve:
sudo: a password is required
Error:
   0: Failed to elevate permissions
   1: Command failed: `/usr/bin/sudo -v`
   2: `/usr/bin/sudo` failed: exit status: 1

Location:
   src/sudo.rs:89

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                 ⋮ 1 frame hidden ⋮
   2: <E as eyre::context::ext::StdError>::ext_report::h79a99fe6a89b8b22
      at <unknown source file>:<unknown line>
   3: <std::process::Command as topgrade::command::CommandExt>::status_checked_with::h526ee16933a66bd4
      at <unknown source file>:<unknown line>
   4: topgrade::sudo::Sudo::elevate::h1892cb5b3644cb70
      at <unknown source file>:<unknown line>
   5: topgrade::run::h85881cb821928fc1
      at <unknown source file>:<unknown line>
   6: topgrade::main::h976145190cd3b7d0
      at <unknown source file>:<unknown line>
   7: std::sys_common::backtrace::__rust_begin_short_backtrace::h150c460b76dc8680
      at <unknown source file>:<unknown line>
   8: main<unknown>
      at <unknown source file>:<unknown line>
   9: __libc_start_call_main<unknown>
      at <unknown source file>:<unknown line>
  10: __libc_start_main_alias_2<unknown>
      at <unknown source file>:<unknown line>
  11: _start<unknown>
      at <unknown source file>:<unknown line>
```

The frame 4 is the root cause:

```
   4: topgrade::sudo::Sudo::elevate::h1892cb5b3644cb70
      at <unknown source file>:<unknown line>
```

As you can see, without the debugging info, it does not provide the file name and line number.

-----

What about setting `debug = true`, this would makes the `topgrade` binary enormously big,  160 MB on my Linux host.

Reply:
Hm, OK. Thanks for catching this, I didn't notice it. I've set the location information to `true` for now.

In the future, maybe we should add a `--backtrace` argument and also turn on location information only if `--backtrace` is given?

Reply:
> In the future, maybe we should add a --backtrace argument and also turn on location information only if --backtrace is given?

Sounds good to me, or maybe we can use the `--verbose` argument here, anyways, both ways are good